language/stdlib/transaction_scripts/add_currency_to_account.move

```diff
@@ -5,10 +5,19 @@ use 0x1::Libra;
 use 0x1::Roles;
 use 0x1::Signer;
 
-/// Add a `Currency` balance to `account`, which will enable `account` to send and receive
-/// `Libra<Currency>`.
-/// Aborts with NOT_A_CURRENCY if `Currency` is not an accepted currency type in the Libra system
-/// Aborts with `LibraAccount::ADD_EXISTING_CURRENCY` if the account already holds a balance in
+/// # Sender Preconditions
+/// The sending account must have a role that allows holding balances (VASP or DD). The
+/// currency being added must be a registerd on-chain currency.
+/// # Accounts Affected
+/// Only the sending account is updated. Transaction fees may be collected.
+/// # Accounts Created
+/// None
+/// # Description
+/// Adds a `Currency` balance to the sending `account`. This will enable `account` to
+/// send, receive, and hold coins in that currency (`Libra<Currency>`).
```

Comment:
```suggestion
/// send, receive, and hold coins in `Libra::Libra<Currency>`.
```

I think the extra `Libra::`makes the doc generator to link to the correct resource.